core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -120,6 +120,7 @@ class KafkaApis(val requestChannel: RequestChannel,
   type FetchResponseStats = Map[TopicPartition, RecordConversionStats]
   this.logIdent = "[KafkaApi-%d] ".format(brokerId)
   val adminZkClient = new AdminZkClient(zkClient)
+  val configHelper = new ConfigHelper(metadataCache, config, replicaManager.configRepository)
```

Comment:
nit: it's a little ugly to reach into `ReplicaManager` in order to get the dependency we need. This also makes the mocking more difficult. Can we instead add `ConfigRepository` as a formal parameter?